docs/data-sources/repository_credential.md

```diff
@@ -0,0 +1,153 @@
+---
+Title: "Repository Credential Data Source"
+Description: |-
+    Creating the Repository Credential data source.
+---
+
+# Repository Credential
+
+The `tanzu-mission-control_repository_credential` data source allows you to get repository credential to a particular scope through Tanzu Mission Control.
+
+[repository-credential]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-657661A2-B26E-412A-9A46-7467A44A075A.html
+
+## repository-credential Scope
```

Comment:
Please start any heading with a capital letter and remove the hyphen in between.